frontend/src/components/buttons/DropdownSelect.styles.js

```diff
@@ -56,6 +56,7 @@ export const DropdownWrapper = styled.div`
         border-radius: 4px;
         box-shadow: 0 3px 5px 1px var(--tonic-grey);
         padding: 5px 3px;
+        z-index: 10;
```

Comment:
Have z indexes been documented anywhere?
I can see problems rising when they are not...

Reply:
I don't think so. Where do you think they should be documented?